src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Controller/ReusedOffSiteSummaryListControllerTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.AatfReturn.Controller
+{
+    using System;
+    using System.Web.Mvc;
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Api.Client;
+    using EA.Weee.Web.Areas.AatfReturn.Controllers;
+    using EA.Weee.Web.Areas.AatfReturn.ViewModels;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Controllers.Base;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+    using FakeItEasy;
+    using FluentAssertions;
+    using Xunit;
+
+    public class ReusedOffSiteSummaryListControllerTests
+    {
+        private readonly IWeeeClient weeeClient;
+        private readonly ReusedOffSiteSummaryListController controller;
+        private readonly BreadcrumbService breadcrumb;
+        private readonly IWeeeCache cache;
+        private readonly IMapper mapper;
+
+        public ReusedOffSiteSummaryListControllerTests()
+        {
+            weeeClient = A.Fake<IWeeeClient>();
+            breadcrumb = A.Fake<BreadcrumbService>();
+            cache = A.Fake<IWeeeCache>();
+            mapper = A.Fake<IMapper>();
+
+            controller = new ReusedOffSiteSummaryListController(() => weeeClient, breadcrumb, cache, mapper);
+        }
+
+        [Fact]
+        public void CheckReuseOffSiteSummaryListControllerInheritsExternalSiteController()
+        {
+            typeof(ReusedOffSiteSummaryListController).BaseType.Name.Should().Be(typeof(ExternalSiteController).Name);
+        }
+
+        [Fact]
+        public async void IndexGet_GivenValidViewModel_BreadcrumbShouldBeSet()
+        {
+            var organisationId = Guid.NewGuid();
+
+            await controller.Index(organisationId, A.Dummy<Guid>(), A.Dummy<Guid>());
+
+            Assert.Equal(breadcrumb.ExternalActivity, BreadCrumbConstant.AatfReturn);
```

Comment:
Add asserts that the ExternalOrganisation and SchemePublicInfo properties are set on the breadcrumb